crowdsourcing/serializers/task.py

```diff
@@ -1,16 +1,11 @@
-__author__ = 'elsabakiu'
+__author__ = ['elsabakiu' , 'megha']
 
 from crowdsourcing import models
-from datetime import datetime
 from rest_framework import serializers
-import json
 
-
-class TaskPropertiesSerializer(serializers.ModelSerializer):
+class TaskSerializer(serializers.ModelSerializer):
     class Meta:
         model = models.Task
+        fields = ('module', 'statuses', 'status', 'deleted', 'created_timestamp', 'last_updated')
+        read_only_fields = ('module', 'statuses', 'created_timestamp', 'last_updated')
 
-
-class TaskWorkerSerializer(serializers.ModelSerializer):
-    class Meta:
-        model = models.TaskWorker
```

Comment:
There is no model called TaskProperties in models.py
I have added TaskWorkerSerializer to serializers/workers.py
I'll add other relevant serialisers for tasks once worker is done.